### DISCUSSION TITLE
本地部署了下，发现公式识别效果比较差

### DISCUSSION THREAD

**Discussion by zju-xiaoyaoyou** [OleehyO/TexTeller#95] [feedback]
直接把huggingface的模型权重下载下来，然后本地启动web服务后，发现公式识别效果挺差的，是开源的模型权重不是最新的吗？

**Comment by zju-xiaoyaoyou** [comment]
![image](https://github.com/user-attachments/assets/057dfd36-3ee5-4f7f-946f-6dd48bb1b1a6)

**Comment by gk966988** [comment]
多试试其他图像，感觉是个例，这个公式识别的整体表现还是可以的
